templates/unbound.conf.erb

```diff
@@ -160,7 +162,7 @@ server:
 <% else -%>
   val-clean-additional: no
 <% end -%>
-<% if @val_permissive_mode -%>
+<% if @val_permissive_mode && !@disable_dnssec -%>
```

Comment:
If both `@val_permissive_mode` and `@disable_dnssec` are true here, than the line for `permissive` doesn't get included in the config.  Is that the desired outcome here?

Reply:
i dont think it makes much difference the removal of the trust anchors file disables dnssec

https://unbound.net/documentation/howto_turnoff_dnssec.html